dotnet/src/Skills/Skills.Web/Google/GoogleConnector.cs

```diff
@@ -10,6 +10,7 @@
 using Microsoft.Extensions.Logging;
 using Microsoft.Extensions.Logging.Abstractions;
 using Microsoft.SemanticKernel.Diagnostics;
+using static Microsoft.SemanticKernel.Skills.Web.Bing.BingConnector;
```

Comment:
Making the GoogleConnector depend on the BingConnector feels odd -- maybe it's best to refactor that class out into it's own. Curious what others may think re: it being public or internal.